src/components/Testing/operate.test.js

```diff
@@ -0,0 +1,16 @@
+import operate from '../../logic/operate';
+
+describe('Testing operate.js', () => {
+  test('Test for add function', () => {
+    expect(operate(3, 5, '+')).toBe('8');
+  });
+  test('Test for subtract function', () => {
+    expect(operate(100, 50, '-')).toBe('50');
+  });
+  test('Test for multiply function', () => {
+    expect(operate(10, 10, 'x')).toBe('100');
+  });
+  test('Test for division function', () => {
+    expect(operate(10, 2, '÷')).toBe('5');
+  });
```

Comment:
- [ ] Great work writing these tests, your work is awesome. 👏 There is just one more operation you didn't test for. That is the modulus operation (`%`). Kindy write tests for the modulus (`%`) operation in this file as the requirements state that all scenarios for the calculate.js and operate.js should be tested. 💻 😄